ui_framework/components/form/_select.scss

```diff
@@ -0,0 +1,12 @@
+/**
+ * 1. Embedded SVG of fa-caret-down (https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/svg/caret-down.svg).
+ * 2. Make room on right side for the caret.
+ */
+.kuiSelect {
+  @include formControl;
+  padding-right: 30px; /* 2 */
+  background-image: url('data:image/svg+xml;utf8,<svg width="1792" height="1792" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M1408 704q0 26-19 45l-448 448q-19 19-45 19t-45-19l-448-448q-19-19-19-45t19-45 45-19h896q26 0 45 19t19 45z"/></svg>'); /* 1 */
```

Comment:
I'm far from an expert on this stuff but I have two questions: 

1. Wouldn't this SVG be re-rendered each time there's a select list on the page, because its inlined via `data:`? Wouldn't a pre-rendered (and potentially cacheable by the browser) image be better?

2. Orthogonal: would it better to use a sprite for performance reasons? I imagine there are other such icons we might use in our styles. This might be premature optimization though so now may not be the right time, particularly if it adds more complexity.